subprojects/packagefiles/glbinding/pch/pch.h

```diff
@@ -0,0 +1 @@
+#include <glbinding/Binding.h>
```

Comment:
About this file...

Upstream has [its own precompiled header](https://github.com/cginternals/glbinding/blob/master/source/glbinding/source/Binding_pch.h) - which is why I've decided to make use of this feature in the first place. However, that header is included by other files (which meson docs warns not to do, although it doesn't seem to break in practice) and has a `#pragma once` guard, which produces a compiler warning. 

As for build time - there's a clear benefit, reducing it from ~47 seconds to ~36 seconds on my machine.